src/Factory/ConfigFactory.php

```diff
@@ -45,12 +45,12 @@ public static function createConfig(Cache\ConfigCache $configCache)
 	}
 
 	/**
-	 * @param Cache\ConfigCache  $configCache The config cache of this adapter
+	 * @param Cache\PConfigCache  $configCache The config cache of this adapter
 	 * @param int                $uid         The UID of the current user
 	 *
 	 * @return Config\PConfiguration
 	 */
-	public static function createPConfig(Cache\ConfigCache $configCache, $uid = null)
+	public static function createPConfig(Cache\PConfigCache $configCache, $uid = null)
```

Comment:
Are you sure about this change? `$configCache->get('system', 'config_adapter')` doesn't seem to be a PConfig call.